microsoft/PowerToys
# Report tool improvements

## Summary of the Pull Request

- skip packaging installers, format time with seconds, remove monitor-info-report tool
- move project to tools 
- move BugReportTool to Tools folder when installed

## Validation Steps Performed
- build installer and install dev version
- run installed report tool and check report file
